frontend/components/styles/Nav.tsx

```diff
@@ -1,68 +1,37 @@
 import React, { useContext } from 'react'
-import s from 'styled-components'
+import Image from 'next/image'
 import Link from 'next/link'
+import { useRouter } from 'next/router'
 
 import { AuthUserContext } from '@/utils/auth'
 import { User } from '@/utils/types'
-import { NAV_WIDTH } from '@/components/styles/sizes'
 import { colors } from '@/components/styles/colors'
-import { InlineText, Text } from '@/components/styles/Text'
-import { Icon } from '@/components/styles/Icons'
-import { Group } from '@/components/styles/Layout'
 import {
   ANALYTICS_ROUTE,
   DASHBOARD_ROUTE,
   SETTINGS_ROUTE,
 } from '@/utils/routes'
 
-const PROFILE_HEIGHT = '24vh'
-const PROFILE_PIC_SIZE = '4rem'
-
-const ProfileWrapper = s.div`
-  height: ${PROFILE_HEIGHT};
-  background-color: ${colors.NAV_PROFILE_BACKGROUND};
-  text-align: center;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-`
-
-const ProfilePicWrapper = s.div`
-  border-radius: 50%;
-  background-color: ${colors.LIGHT_GRAY};
-  width: ${PROFILE_PIC_SIZE};
-  height: ${PROFILE_PIC_SIZE};
-  margin: 0 auto;
-`
+const iconSrcMap: { [key: string]: string } = {
+  dashboard: '/icons/dashboard.svg',
+  analytics: '/icons/analytics.svg',
+  settings: '/icons/settings.svg',
+}
 
 const Profile = ({ user }: { user: User }) => {
   return (
-    <ProfileWrapper>
-      <Group center>
-        <ProfilePicWrapper>
-          <InlineText bold heading style={{ lineHeight: PROFILE_PIC_SIZE }}>
-            {user.first_name[0].toUpperCase()}
-            {user.last_name[0].toUpperCase()}
-          </InlineText>
-        </ProfilePicWrapper>
-        <Text bold heading>
-          {user.first_name} {user.last_name}
-        </Text>
-      </Group>
-    </ProfileWrapper>
+    <div className="flex items-center gap-2 font-work-sans">
```

Comment:
was there a reason you didn't fully translate everything like height adn background color for example?

Reply:
A lot of these things are purely aesthetic changes - mostly for styling consistency. I want to make visual changes just to test things first.